### ISSUE TITLE
Fix colour contrast error in tabs section

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#283]
with our new localgov_sa11y module, we're getting a (false) error reported that the colour contrast is failing on the active tab of our tabs. 

The green / white combination is 5.13:1 which passes AA for small text. 

However the plugin seems to think it's lightgrey/white 

![Image](https://user-images.githubusercontent.com/3852805/192269194-7daf30fd-d022-49f1-b486-6de56c9f9cb2.png)

**Comment by markconroy** [comment]
I'll close this one https://github.com/localgovdrupal/localgov_base/issues/282

**Comment by msayoung** [comment]
Actually, I realise it's the .visually-hidden H2 which is causing the error, not the active tab, which is annoying but easily fixed.

**Comment by msayoung** [comment]
https://github.com/localgovdrupal/localgov_base/pull/284
